src/app/endpoint/edition/commit.rs

```diff
@@ -156,25 +156,27 @@ impl RequestHandler for CommitHandler {
     }
 }
 
-#[derive(Serialize)]
-struct EditionCommitNotification {
-    status: &'static str,
+#[derive(Serialize, Deserialize)]
+pub(crate) struct EditionCommitNotification {
+    pub status: String,
```

Comment:
```suggestion
    pub(crate) status: String,
```
Maybe these fields should also be `pub(crate)`, like in the structure above?

Reply:
у нас весь тип pub(crate), зачем при этом делать pub(crate) на поле в типе?